docs/wfirst.rst

```diff
@@ -0,0 +1,96 @@
+******************
+WebbPSF for WFIRST
+******************
+
+
+
+.. figure:: ./wfirst_figures/webbpsf-wfirst_page_header.png
+   :align: center
+   :alt: Sample PSFs for the filters in the WFIRST WFI.
+
+   Sample PSFs for the filters in the WFIRST WFI. Angular scale in arcseconds, log-scaled intensity.
+
+Introduction
+============
+
+WebbPSF provides a framework for JWST instrument PSF calculations that is easily extensible to other instruments and observatories. The :py:mod:`webbpsf.wfirst` module was developed to enable simulation of WFIRST's Wide Field Instrument (WFI) based on reference information from Goddard Space Flight Center.
+
+At this time, the only instrument simulated is the WFI, but that may change in the future. To work with the WFI model, import and instantiate it as follows::
+
+>>> import webbpsf
+>>> from webbpsf import wfirst
+>>> wfi = wfirst.WFI()
+
+Usage of the WFI model class is, for the most part, just like any other WebbPSF instrument model. For help setting things like filters, position offsets, and sampling refer back to :ref:`using_api`.
+
+What is different (and, for now, only available for the WFIRST WFI) is the support for field dependent aberrations. With as large a field of view as the WFI is designed to cover, there will be variation in the PSF from one end of the field of view to the other. WebbPSF's WFI model faithfully reproduces the field dependent aberrations calculated from the Goddard WFIRST team's simulated WFI design, leaving it to the user to decide if their use case requires accurate knowledge of very small variations in the PSF.
+
```

Comment:
That this document will not be the one to tell you if inter-SCA and intra-SCA differences are "big", since we don't know what "big" means for a particular user